docs/guides/self_hosting.md

```diff
@@ -115,3 +115,29 @@ Below are the configurations to specify in the `values_sample.yaml` file.
 | config.values.llm_engine.cache_redis_url | The full url for the redis cluster you wish to connect | Yes |
 | config.values.llm_engine.s3_file_llm_fine_tuning_job_repository | The S3 URI for the S3 bucket/key that you wish to save fine-tuned assets | Yes |
 | config.values.datadog_trace_enabled | Whether to enable datadog tracing, datadog must be installed in the cluster | No |
+
+## Play With It
+Once `helm install` succeeds, you can forward port 5000 from a `llm-engine` pod and test sending requests to it.
+
+First, see a list of pods in the namespace that you performed `helm install` in:
+```
+$ kubectl get pods -n <NAMESPACE_YOU_INSTALL_LLM_ENGINE>
```

Comment:
nit: grammar, maybe something like `NAMESPACE_WHERE_LLM_ENGINE_IS_INSTALLED`  or just `LLM_ENGINE_NAMESPACE`. This should probably just follow whatever helm variable you're using.